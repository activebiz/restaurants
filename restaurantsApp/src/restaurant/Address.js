import React from 'react';
import {View, Text} from 'react-native';

const Address = props => {
  const {streetAddress, addressLocality, postalCode} = props;
  return (
    <View>
      <Text>{streetAddress}</Text>
      <Text>{addressLocality}</Text>
      <Text>{postalCode}</Text>
    </View>
  );
};

export default Address;
