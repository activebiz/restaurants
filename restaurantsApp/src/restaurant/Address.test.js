import 'react-native';
import React from 'react';
import Address from './Address';
import renderer from 'react-test-renderer';

describe('Address', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<Address />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
