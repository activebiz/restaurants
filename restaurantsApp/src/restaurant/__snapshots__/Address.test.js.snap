// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Address renders correctly 1`] = `
<View>
  <Text />
  <Text />
  <Text />
</View>
`;
